libcontainer/cgroups/fs/cpuacct.go

```diff
@@ -83,8 +83,8 @@ func (s *CpuacctGroup) GetStats(path string, stats *cgroups.Stats) error {
 
 // Returns user and kernel usage breakdown in nanoseconds.
 func getCpuUsageBreakdown(path string) (uint64, uint64, error) {
-	userModeUsage := uint64(0)
-	kernelModeUsage := uint64(0)
+	var userModeUsage uint64
+	var kernelModeUsage uint64
```

Comment:
```suggestion
	var userModeUsage, kernelModeUsage uint64
```

Reply:
Thanks @kolyshkin . Have made the changes.